inc/class-upgrade.php

```diff
@@ -997,8 +997,13 @@ private function upgrade_205() {
 	/**
 	 * Performs the 20.7 upgrade routine.
 	 * Removes the metadata related to the settings page introduction modal for all the users.
+	 * Also, schedules another cleanup scheduled action.
 	 */
 	private function upgrade_207() {
```

Comment:
We're probably going to roll this out with 20.8, so probably best to fix the upgrade routine now, so as to not forget.